src/online_managers.rs

```diff
@@ -179,7 +179,7 @@ pub struct LoginResponseUser {
 }
 
 #[derive(Deserialize)]
-pub struct LoginResponse {
+pub(crate) struct LoginResponse {
```

Comment:
Are you sure this is not used by customers of the lib?

A good check is to try to compile libetebase and etebase-py with this version.

Reply:
Ditto.

Reply:
Can you try compiling libetebase (the C library) to just make sure nothing broke?

Reply:
Yep, etebase-py, etebase-java and libetebase still build.